.vscode/settings.json

```diff
@@ -1,12 +1,17 @@
 {
     "json.schemas": [
         {
-            "fileMatch": ["*.scene.json"],
+            "fileMatch": [
+                "*.scene.json"
+            ],
             "url": "./schemas/scene.schema.json"
         },
         {
-            "fileMatch": ["*.register.json"],
+            "fileMatch": [
+                "*.register.json"
+            ],
             "url": "./schemas/register.schema.json"
         }
-    ]
-}
+    ],
+    "files.insertFinalNewline": true
+}
```

Comment:
Interesting, adding a setting that ensures final newline did not add final new line

![rah3of6xp3531](https://github.com/membraneframework/video_compositor/assets/9753141/385c405c-d2a5-46fc-a992-6968ca6a305f)

Reply:
either way, no strong opinion, but I don't think this should be here. It should be in your personal config not the project one